docs/cloud.rst

```diff
@@ -0,0 +1,201 @@
+.. _cloud:
+
+Pangeo Cloud
+============
+
+Pangeo cloud is an experimental service providing cloud-based data-science
+environments.
+
+.. warning::
+    We currently have no long-term funding to support Pangeo Cloud.
+    At present, cobble together funding from various grants from funding
+    agencies and cloud providers.
+    All users must understand that Pangeo Cloud environments could disappear
+    at any time.
+
+Sign Up
+-------
+
+Pangeo Cloud is available to researchers across the world engaged in
+data-intensive Earth and Environmental Science.
+Prospective users must apply using the following form:
+
+- `Pangeo Cloud Application`_
+
+Access will be granted based on a review of the proposed research project
+and its technical suitability for Pangeo Cloud.
+
+.. _Pangeo Cloud Application: https://forms.gle/J3hVVBgobwpYVWHF8
+
+Clusters
+--------
+
+Pangeo Cloud currently comprises two different computing clusters:
+
+- `us-central1-b.gcp.pangeo.io <https://us-central1-b.gcp.pangeo.io/>`_: A cluster in
+  Google Cloud Platform.
+- `uswest2.pangeo.io <https://uswest2.pangeo.io>`_: A cluster in AWS.
+
+Once your application is approved, you will be able to log in to the resources
+you requested.
```

Comment:
A "how do I choose" section might be helpful (perhaps with a link to https://github.com/pangeo-data/multicloud-demo, if we think that dataset location is the determining factor).